src/configuration.ts

```diff
@@ -90,7 +90,7 @@ const BUILT_IN_CONFIG = /^tslint:(.*)$/;
  * of the search for a configuration.
  * @returns Load status for a TSLint configuration object
  */
-export function findConfiguration(configFile: string | null, inputFilePath: string): IConfigurationLoadResult {
+export function findConfiguration(configFile: string | null, inputFilePath?: string): IConfigurationLoadResult {
```

Comment:
to prevent call like `findConfiguration(null, undefined);` this function needs overloads. just add these lines above:

```ts
export function findConfiguration(configFile: string | null, inputFilePath: string): IConfigurationLoadResult;
export function findConfiguration(configFile: string, inputFilePath?: string): IConfigurationLoadResult
```

Reply:
Good idea! For some reason I didn't even think of overloads here 🙂